onward/app/feed/MostPopularAgent.scala

```diff
@@ -59,8 +61,15 @@ class GeoMostPopularAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi
         item: JsValue <- ophanResults.asOpt[JsArray].map(_.value).getOrElse(Nil)
         url <- (item \ "url").asOpt[String]
       } yield {
-        contentApiClient.getResponse(contentApiClient.item(urlToContentPath(url), Edition.defaultEdition))
-          .map(_.content.map(RelatedContentItem(_)))
+        contentApiClient
+          .getResponse(contentApiClient
+            .item(urlToContentPath(url), edition)
+            .showTags("paid-content")
+            .showSection(true)
+            .showFields("isAppropriateForSponsorship"))
```

Comment:
Actually looking at a capi response, the field is called `isInappropriateForSponsorship`